flink-connector/flink/src/main/java/org/apache/gravitino/flink/connector/utils/TypeUtils.java

```diff
@@ -36,6 +36,12 @@
 
 public class TypeUtils {
 
+  // Flink supports timestamp precision from 0 to 9 (nanosecond precision).
+  protected static final int FLINK_SECONDS_PRECISION = 0;
+  protected static final int FLINK_MILLIS_PRECISION = 3;
+  protected static final int FLINK_MICROS_PRECISION = 6;
+  protected static final int FLINK_NANOS_PRECISION = 9;
```

Comment:
Is there any Flink official doc description about this? If so, please attach to the code comment, thanks!

Reply:
Thanks for your time, comments added.